Blocks/MMQ/test.cc

```diff
@@ -282,8 +299,9 @@ TEST(InMemoryMQ, MMPQAllowsTimeExplicitlyGoingBack) {
 
   // Publish "two", "three", and "four". The first one to get published is "three", but, as it's published
   // into the future, it won't get processed until "four" is. Note the index of "two" is `3`, not `2`.
```

Comment:
Fix the comment as `PublishIntoTheFuture` is deprecated? Not sure how to phrase it better, but it should mention `UpdateHead` imho.